### DISCUSSION TITLE
Raspotify crashes as soon as i start playing music from spotify with status=1/FAILURE

### DISCUSSION THREAD

**Discussion by dereckchiu** [dtcooper/raspotify#614] [Q&A]
I'm not sure what I did wrong, but here are the logs, I'm trying to play raspotify through pulseaudio, some help will be very much appreciated!

ubuntu@ubuntu:~$ journalctl -u raspotify -b
Mar 14 05:34:54 ubuntu systemd[1]: Started Raspotify (Spotify Connect Client).
Mar 14 05:35:05 ubuntu librespot[762]: [2023-03-14T05:35:05Z WARN  librespot_core::apresolve] Ignoring blacklisted access point ap-gew4.spotify.com:443
Mar 14 05:37:58 ubuntu librespot[762]: Failed to create secure directory (//.config/pulse): No such file or directory
Mar 14 05:37:58 ubuntu librespot[762]: Failed to load cookie file from cookie: No such file or directory
Mar 14 05:37:58 ubuntu librespot[762]: Failed to create secure directory (//.config/pulse): No such file or directory
Mar 14 05:37:58 ubuntu librespot[762]: Failed to load cookie file from cookie: No such file or directory
Mar 14 05:37:58 ubuntu librespot[762]: ALSA lib pulse.c:242:(pulse_connect) PulseAudio: Unable to connect: Access denied
Mar 14 05:37:58 ubuntu librespot[762]: [2023-03-14T05:37:58Z ERROR librespot_playback::player] Audio Sink Error Connection Refused: <AlsaSink> Device def>
Mar 14 05:37:58 ubuntu systemd[1]: raspotify.service: Main process exited, code=exited, status=1/FAILURE
Mar 14 05:37:58 ubuntu systemd[1]: raspotify.service: Failed with result 'exit-code'.
Mar 14 05:37:58 ubuntu systemd[1]: raspotify.service: Triggering OnFailure= dependencies.

**Comment by JasonLG1979** [comment]
It looks like you're trying to run Raspotify on a desktop system. That is not supported. Raspotify is designed to be run on headless systems as stated in the [readme](https://github.com/dtcooper/raspotify#readme) and [basic setup guide](https://github.com/dtcooper/raspotify/wiki/Basic-Setup-Guide). The reason being is that Raspotify is a system level service and pulseaudio and/or pipewire are user level services. They are not compatible with the Raspotify service.

Your options are either to run pulseaudio as a system level service as described [here](https://github.com/dtcooper/raspotify/wiki/Raspotify-with-PulseAudio-as-a-system-service) or disable the Raspotify service and write your own systemd unit file to create your own user level service with the provided librespot binary. Both use cases are unsupported so you're on your own.

**Reply by dereckchiu** [reply]
Hi Jason, thanks for your reply, I'm running raspotify on Ubuntu server 18.04, that being said, is the solution still to run pulseaudio as a system level service? Thanks for the reply.

**Reply by JasonLG1979** [reply]
> I'm running raspotify on Ubuntu server 18.04,

Ubuntu 18.04 is crazy old. I'm surprised it actually installs at all.

> is the solution still to run pulseaudio as a system level service?

On a server the solution is to uninstall PulseAudio if you can. Other then that, as I said you're on your own. That page is a user contribution.

**Reply by dereckchiu** [reply]
Thanks Jason!
